src/App.css

```diff
@@ -7,6 +7,14 @@
   border-bottom: 1px solid #d8d8d8;
 }
 
```

Comment:
- [ ] it would be better if the app does not overflow the size of the window. Kindly fix this minor issue.

![tempsnip](https://user-images.githubusercontent.com/64173189/156856389-92302766-99dd-4a54-a820-70752f27dcd8.png)

Reply:
Thank you! I have fixed the overflow.